monitoring/voltage-monitoring/src/test/resources/networkKO.xiidm

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I believe this is the same network as the one used for AngleMonitoring. If so, it would be better if you can find a way to use the same network rather than duplicate it.

Reply:
The standard network used for VoltageMonitoring and AngleMonitoring (network.xiidm) is already duplicated (the file network.xiidm in angle-monitoring/src/test/resources/ is the exact same as the one in voltage-monitoring/src/test/resources/) so I though that I would do the same for the networkKO.xiidm file.

Reply:
Okay i see. This duplication will probably disappear when the 2 algorithms will be merged.